env/env.go

```diff
@@ -42,6 +43,17 @@ var envVars map[string]string
 
 var currentEnv = "default"
 
+// GetSpecDir returns the specification directory.
+// It checks whether the environment variable for gauge_specs_dir is set.
+// It returns 'specs' otherwise
+func GetSpecDir() string {
```

Comment:
Would be better if this was moved to util.go.

Reply:
fine with me